src/appservice.ts

```diff
@@ -38,6 +38,10 @@ export function getAppservice(config: BridgeConfig, registration: IAppserviceReg
         },
         storage: storage,
         intentOptions: {
+            // If encryption support is enabled, we cannot expire Intent objects or we risk
+            // a resource contention on the Sled DB.
+            maxCached: config.encryption && 0,
+            maxAgeMs: config.encryption && 0,
```

Comment:
Better use something like `config.encryption ? foo : 0` to limit what falsey values are passed to the Intent.
```
> true && 0
0
> false && 0
false
> null && 0
null
> undefined && 0
undefined
```

Reply:
Don't we trust `config.encryption` to be object|undefined though, as that's strongly typed in the config.